pkg/sql/ttl/ttljob/ttljob.go

```diff
@@ -127,17 +248,49 @@ func (t rowLevelTTLResumer) Resume(ctx context.Context, execCtx interface{}) err
 		if ttl == nil {
 			return errors.Newf("unable to find TTL on table %s", desc.GetName())
 		}
+		ttlSettings = *ttl
 
 		ranges, err = kvclient.ScanMetaKVs(ctx, txn, desc.TableSpan(p.ExecCfg().Codec))
 		if err != nil {
 			return err
 		}
-		ttlSettings = *ttl
+
+		_, dbDesc, err := descs.GetImmutableDatabaseByID(
+			ctx,
+			txn,
+			desc.GetParentID(),
+			tree.CommonLookupFlags{
+				Required: true,
+			},
+		)
+		if err != nil {
+			return err
+		}
+		schemaDesc, err := descs.GetImmutableSchemaByID(
+			ctx,
+			txn,
+			desc.GetParentSchemaID(),
+			tree.CommonLookupFlags{
+				Required: true,
+			},
+		)
+		if err != nil {
+			return err
+		}
+
+		tn := tree.MakeTableNameWithSchema(
+			tree.Name(dbDesc.GetName()),
+			tree.Name(schemaDesc.GetName()),
+			tree.Name(desc.GetName()),
+		)
+		name = tn.FQString()
 		return nil
 	}); err != nil {
 		return err
 	}
 
+	metrics := p.ExecCfg().JobRegistry.MetricsStruct().RowLevelTTL.(*RowLevelTTLAggMetrics).loadMetrics(name)
```

Comment:
do we have the pattern of keying metrics by table name elsewhere? what if the table name changes?

actually i may not understand what `loadMetrics` does

Reply:
yeah it converts an aggmetric with no label attached to a simple metric which does.

Reply:
See other usages of aggmetric, from memory slimetrics was an example.